### PR TITLE
Week one in class homework

### PR DESCRIPTION
**Pull request by danielwilliams12**

### CODE REVIEW

File: week1/exercises/roster.txt [hunk 1]
```diff
@@ -1,4 +1,17 @@
 #, name, email, github, twitter, hip chat 
+<<<<<<< HEAD
```
Comment (reneedv) [inline week1/exercises/roster.txt:2]: These symbols are what you see when there is a merge conflict.

File: week2/homework/simon_says_spec.rb [hunk 2]
```diff
@@ -14,34 +14,34 @@
   end
 
   it "should shout hello" do
-    shout("hello").should == "HELLO"
+    shout("HELLO").should == "HELLO"
```
Comment (reneedv) [inline week2/homework/simon_says_spec.rb:17]: The idea here is to make the tests pass by writing your code to match this spec. You are not supposed to modify the tests.

File: week2/homework/simon_says.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+module SimonSays
```
Comment (reneedv) [inline week2/homework/simon_says.rb:1]: you will want to use string methods here to make the specs pass as they were originally written.

File: week1/exercises/roster.txt (hunk 1 above)
Comment (danielwilliams12) [inline week1/exercises/roster.txt:2]: Thanks

I got the idea tonight in class

Will correct

Dan Williams, PMP, PMI-ACP, CSP

Cell: 425-8776333

שָׁלוֹם עֲלֵיכֶם‎

peace to you

File: week2/homework/simon_says.rb (hunk 3 above)
Comment (danielwilliams12) [inline week2/homework/simon_says.rb:1]: How can I get a copy of the original test?

thanks

Dan Williams, PMP, PMI-ACP, CSP

Cell: 425-8776333

שָׁלוֹם עֲלֵיכֶם‎

peace to you

**Pull request closed without merging**